portal/src/it/scala/cromwell/pipeline/controller/ProjectControllerItTest.scala

```diff
@@ -0,0 +1,86 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.dimafeng.testcontainers.{ForAllTestContainer, PostgreSQLContainer}
+import com.typesafe.config.Config
+import cromwell.pipeline.ApplicationComponents
+import cromwell.pipeline.datastorage.dto.Project
+import cromwell.pipeline.datastorage.dto.project.{ProjectDeleteRequest, ProjectUpdateRequest}
+import cromwell.pipeline.utils.auth.{AccessTokenContent, TestContainersUtils, TestProjectUtils, TestUserUtils}
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+import org.scalatest.{AsyncWordSpec, Matchers}
+
+class ProjectControllerItTest
+    extends AsyncWordSpec
+    with Matchers
+    with ScalatestRouteTest
+    with PlayJsonSupport
+    with ForAllTestContainer {
+
+  override val container: PostgreSQLContainer = TestContainersUtils.getPostgreSQLContainer()
+  container.start()
+  implicit val config: Config = TestContainersUtils.getConfigForPgContainer(container)
+  private val components: ApplicationComponents = new ApplicationComponents()
+  override protected def beforeAll(): Unit = components.datastorageModule.pipelineDatabaseEngine.updateSchema()
+  val ownerId = "1"
+
+  import components.controllerModule.projectController
+  import components.datastorageModule.projectRepository
+  import components.datastorageModule.userRepository
+
+  "ProjectController" when {
+    "getProjectByName" should {
+      "return a project with the same name" in {
+        val dummyUser = TestUserUtils.getDummyUser(ownerId)
+        val dummyProject = TestProjectUtils.getDummyProject("11", ownerId)
```

Comment:
You don't check projectId("11") in your tests, let it generate itself

Reply:
And every time you create dummyUser, but you need only one in DB

Reply:
Done